.github/workflows/cnvm-ci.yml

```diff
@@ -1,6 +1,8 @@
 name: CNVM-CI
 
 on:
+  merge_group:
+    branches: ['main']
   pull_request_target:
```

Comment:
please move every `pull_request_target` to post-merge instead of adding them to the merge queue